python/lsst/ts/MTDomeTrajectory/dome_trajectory.py

```diff
@@ -288,15 +272,12 @@ async def move_dome_elevation(self, desired_dome_elevation):
 
             # Move the dome elevation axis and wait for the target event
             # and the first motion event (so motion has started).
-            self.log.debug("Move dome elevation")
-            if not flushed_motion_event:
-                self.dome_remote.evt_elMotion.flush()
             self.dome_remote.evt_elTarget.flush()
+            self.log.debug("Start a dome elevation move")
```

Comment:
Would you be willing to consider making the log message "Start dome elevation motion" to be in line with line 264?